src/components/screens/monitor/delegates/overview/forgingDetails.js

```diff
@@ -125,7 +125,8 @@ const ForgingDetails = ({
                   <h6>{t('Blocks forged')}</h6>
                   <ProgressBar forgedInRound={forgedInRound} theme={theme} />
                   <p className={styles.blue}>
-                    {`${forgedInRound} / `}
+                    <span className="blocksForged">{`${forgedInRound} `}</span>
```

Comment:
@eniolam1000752 There seems to be some unnecessary space after `forgedInRound` in the object literal